vortexasdk/endpoints/timeseries_result.py

```diff
@@ -0,0 +1,35 @@
+import os
+from multiprocessing.pool import Pool
+from typing import List
+
+import pandas as pd
+
+from vortexasdk.api.search_result import Result
+from vortexasdk.api.timeseries_item import TimeSeriesItem
+
+
+class TimeSeriesResult(Result):
+    """Container class that holds the result obtained from calling a time series endpoint."""
+
+    def to_list(self) -> List[TimeSeriesItem]:
+        """Represents time series as a list."""
+        list_of_dicts = super().to_list()
+        with Pool(os.cpu_count()) as pool:
+            return list(pool.map(TimeSeriesItem.from_dict, list_of_dicts))
+
+    def to_df(self, columns=None) -> pd.DataFrame:
```

Comment:
`columns` arg never appears to be used.  Is it required here?

Reply:
I can see the base class abstract method has it in its signature, is it worth adding it to the return statement's signature?